### DISCUSSION TITLE
video support

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3629] [Mobile] [answered]
in the comments, allow:
* attaching
* sharing
* and viewing videos

this is how currently videos are displayed - just a name of the file

<img width="300" alt="image" src="https://user-images.githubusercontent.com/39104740/112762334-cb01dd80-9007-11eb-95d4-08230398d121.png">

**Comment by candyho** [comment, marked as answer]
Hey, thanks for sharing the feedback. This is something already on our radar, please stay tuned!

**Reply by xxksa2010** [reply]
Hilp
